components/team-info/sections/DocumentsSection.vue

```diff
@@ -72,7 +72,7 @@ const menuOptions: MenuOption[] = [
     label: 'Minute books',
     key: 'minute-books',
     children: documents.value
-      ?.filter((doc) => doc.title?.startsWith('minute-book'))
+      ?.filter((doc) => doc.title?.startsWith('m'))
```

Comment:
Why?

Reply:
a takto je to jednoduchsie ked len na m a r zacinaju subory